src/encoding.md

```diff
@@ -688,6 +689,68 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+
+[ex-mime-from-filename]: #ex-mime-from-filename
+<a name="ex-mime-from-filename"></a>
+## Get MIME type from filename
+
+[![mime-badge]][mime] [![cat-encoding-badge]][cat-encoding]
+
+The following example shows how to return the correct MIME type from a given
+filename using the [mime] crate.
+
+```rust
+# #[macro_use]
```

Comment:
this example does not need to use:
- error_chain
- std::io
- macros
Feel free to remove lines 704-713